package.json

```diff
@@ -132,6 +137,7 @@
     "eslint-plugin-sort-destructure-keys": "^1.3.4",
     "export-to-csv": "^0.2.1",
     "firebase-admin": "^8.12.1",
+    "google-gax": "^2.3.1",
```

Comment:
I needed to add this to fix the weird credential issue I was having with `uploadMocks`